src/code/sys_matrix.c

```diff
@@ -740,7 +742,7 @@ void func_800D1FD4(MtxF* mf) {
 }
 
 /**
- * Gets the rotation the specified matrix represents, using Euler YXZ.
+ * Gets the rotation the specified matrix represents, using Tait-Bryan YXZ angles.
  * The flag value doesn't matter for a rotation matrix. Not 0 does extra calculation.
  */
```

Comment:
It would be nice to clarify that the middle angle (`rotDest->x`) will range from `-0x4000` to `0x4000` (+/- 90 degrees), whereas the other two angles have the full +/- 180 degrees of range. 

This property is generally true of all Euler angle representations, but it might not be obvious to someone who is decomping.